### DISCUSSION TITLE
1.32.1

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#18248] [General]
# Fixes
- [#17961](https://github.com/Koenkk/zigbee2mqtt/issues/17961) Fix `Expected one of: 1, 2, got: '29146'` error (@Koenkk)
- [#5946](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5946) Fix various Eco-Dim.07/Eco-Dim.10 variants not recognised (@KDanielK)
- [#18184](https://github.com/Koenkk/zigbee2mqtt/issues/18184) Fix `Cannot read properties of null (reading 'from')` error for some TuYa devices (@Koenkk)
- [#5727](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5727) Add TuYa exports to `legacy.ts` (@Koenkk)
- [#18111](https://github.com/Koenkk/zigbee2mqtt/issues/18111) Support colors for ADEO BD05C-FL-21-G-ENK (@Koenkk)
- [#18168](https://github.com/Koenkk/zigbee2mqtt/issues/18168) Fix setting color temperature for groups with non color temperature lights not working (@Koenkk)
- [#18169](https://github.com/Koenkk/zigbee2mqtt/issues/18169) Fix `is not a number, got string` errors (@Koenkk)

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.32.1'>1.32.1</a>.</em>

**Comment by kgiedrius** [comment]
Updated to this release in Home assistant and problems with "string and integers" started to appear for all  Gledopto LED controllers :( 
Whole zigbee network stopped working whatever i did, even restarted HA several times. So it's not stable release.

Reverted to 1.31 - and everything works fine.
